src/ILCompiler.TypeSystem/tests/RuntimeDeterminedTypesTests.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Licensed to the.NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.TypeSystem;
+
+using Xunit;
+
+namespace TypeSystemTests
+{
+    public class RuntimeDeterminedTypesTests
+    {
+        private TestTypeSystemContext _context;
+        private ModuleDesc _testModule;
+
+        private MetadataType _referenceType;
+        private MetadataType _otherReferenceType;
+        private MetadataType _structType;
+        private MetadataType _otherStructType;
+        private MetadataType _genericReferenceType;
+        private MetadataType _genericStructType;
+        private MetadataType _genericReferenceTypeWithThreeParams;
+        private MetadataType _genericStructTypeWithThreeParams;
+
+        public RuntimeDeterminedTypesTests()
+        {
```

Comment:
@MichalStrehovsky  I've updated the comment for you. You can push the "pencil" icon and it allows you to edit any comment and also see what was in there literally.